templates/ssh_config.erb

```diff
@@ -109,3 +109,6 @@ GSSAPIDelegateCredentials <%= @ssh_gssapidelegatecredentials %>
 <% if @ssh_config_user_known_hosts_file -%>
   UserKnownHostsFile <%= @ssh_config_user_known_hosts_file.join(' ') %>
 <% end -%>
+<% if @ssh_config_verify_host_key_dns != nil -%>
```

Comment:
This needs spec tests. They will be similar to https://github.com/ghoneycutt/puppet-module-ssh/commit/7c96719526b5c1fe9bbd1d1213cd766a71ee6c92